### PR TITLE
Sets 'Working Directory' according to Visual Studio's Configuration Properties

### LINKED ISSUE #43: Specify Test Execution Working Directory
As described by [Roel_v](https://visualstudiogallery.msdn.microsoft.com/site/search?f%5B0%5D.Type=User&f%5B0%5D.Value=Roel_v)'s comment '[Set working directory](https://visualstudiogallery.msdn.microsoft.com/5f4ae1bd-b769-410e-8238-fb30beda987f#18625)', the working directory of test execution should be explicit and, ideally, defaults to the working directory as specified within a project's property sheets.

### PR DESCRIPTION
**Pull request by netspiri**
Sets 'Working Directory' according to Visual Studio's Configuration Properties or runsettings xml file

Closes #43

### CODE REVIEW

File: BoostTestAdapterNunit/VisualStudio2012AdapterTest.cs [hunk 1]
```diff
@@ -31,6 +31,8 @@ public void SetUp()
         private IProject Project { get; set; }
         private const string DefaultProjectName = "SampleProject";
         private const string DefaultOutput = "test.boostd.exe";
+        private const string DefaultWorkingDirectory = "$(ProjectDir)";
+        private const string DefaultEvaluatedWorkingDirectory = "c:\\";
```
Comment (guwirth) [inline BoostTestAdapterNunit/VisualStudio2012AdapterTest.cs]: Is `c:\` the right default? Or should it not better be something like `c:\temp`?

File: VisualStudioAdapter/IProjectConfiguration.cs [hunk 2]
```diff
@@ -14,11 +14,16 @@ public interface IProjectConfiguration
         /// Determines the fully-qualified path of the project's primary output
         /// </summary>
         string PrimaryOutput { get; }
-
+
```
Comment (guwirth) [inline VisualStudioAdapter/IProjectConfiguration.cs]: formatting: remove blanks

**Comment by guwirth** [comment]
:+1: except some smaller issues (see comments in code)

**Comment by netspiri** [comment]
Hi @guwirth the branch has been modified accordingly as well as squashed for merge.  Thanks.

**Comment by guwirth** [comment]
:+1:

**Pull request merged**